### ISSUE TITLE
SmartCity-Accessibility #61 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#86]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 112 lines...]
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: Dec 20, 2016
JxMaps license valid.
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 510.145 sec <<< FAILURE! - in smartcity.accessibility.navigation.NavigationTest
displayMap(smartcity.accessibility.navigation.NavigationTest)  Time elapsed: 502.294 sec  <<< ERROR!
com.teamdev.jxmaps.internal.internal.ipc.v: Failed to get Browser browserChannel 0
	at com.teamdev.jxmaps.internal.Browser.a(SourceFile:344)
	at com.teamdev.jxmaps.internal.Browser.<init>(SourceFile:181)
	at com.teamdev.jxmaps.internal.Browser.<init>(SourceFile:153)
	at com.teamdev.jxmaps.internal.Browser.<init>(SourceFile:136)
	at com.teamdev.jxmaps.MapBrowser.<init>(SourceFile:31)
	at com.teamdev.jxmaps.swing.MapView.<init>(SourceFile:66)
	at com.teamdev.jxmaps.swing.MapView.<init>(SourceFile:28)
	at smartcity.accessibility.navigation.JxMapsConvertor.<init>(JxMapsConvertor.java:25)
	at smartcity.accessibility.navigation.JxMapsConvertor.displayRoute(JxMapsConvertor.java:58)
	at smartcity.accessibility.navigation.NavigationTest.displayMap(NavigationTest.java:81)


Results :

Tests in error: 
  NavigationTest.displayMap:81 » v Failed to get Browser browserChannel 0

Tests run: 17, Failures: 0, Errors: 1, Skipped: 3

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 09:02 min
[INFO] Finished at: 2016-12-20T14:44:53+00:00
[INFO] Final Memory: 24M/179M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [yaelAmitay] 1b1fd3728c06fa1118bc63466a3bb10e3fa9341d - test display route in map #81

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/61/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
